src/atom-environment.coffee

```diff
@@ -885,8 +885,6 @@ class AtomEnvironment extends Model
         else
           @project.addPath(pathToOpen)
 
-      @applicationDelegate.addRecentDocument(pathToOpen)
-
```

Comment:
This method is only called when the browser process tells a render process to open a file (e.g. when initially opening a file from the terminal, or using a special command like `Application:  Open Your Stylesheet). It seems like we should add to the recent documents list *whenever* we open a file.